src/cpu/probes/LoopPoint.py

```diff
@@ -0,0 +1,13 @@
+from m5.params import *
+from m5.objects.Probe import ProbeListenerObject
+
+
+class LoopPoint(ProbeListenerObject):
+
+    type = "LoopPoint"
+    cxx_header = "cpu/probes/looppoint.hh"
+    cxx_class = "gem5::LoopPoint"
+
+    target_pc = VectorParam.UInt64(0, "the target PC")
+    core = Param.BaseCPU(NULL, "the connected cpu")
```

Comment:
Also, what does it mean to be NULL here? Is that valid? If not, then we should just have no default.